packages/gui/src/components/nfts/gallery/NFTGallery.tsx

```diff
@@ -90,6 +92,10 @@ export default function NFTGallery() {
         return false;
       }
 
+      if (hideObjectionableContent && sensitiveContentObject[nft.$nftId]) {
```

Comment:
Does `sensitiveContentObject` need to be included in the useEffect dependency array?

In my testing, I'm seeing `sensitive-content: true` NFTs appearing in the gallery when I have 'Hide objectionable content' set to true. If I set it to false, and then back to true, the gallery hides the `sensitive-content: true` NFT.

I can repro this easily by having the gallery open and minting an NFT with sensitive-content set to true. The NFT will appear in the gallery when it should be hidden.